### PR TITLE
Document remaining non-scrolling symbols

### PR DESCRIPTION
**Pull request by abarth**
This patch finishes documenting all the symbols except for those related to
scrolling.

### CODE REVIEW

File: packages/flutter/lib/src/material/tabs.dart [hunk 1]
```diff
@@ -915,6 +915,7 @@ class _TabBarViewState extends State<TabBarView> {
 ///
 /// Used by [TabPageSelector] to indicate the selected page.
 class TabPageSelectorIndicator extends StatelessWidget {
+  /// Creates an indicator used by [TabPageSelector].
```
Comment (Hixie) [inline packages/flutter/lib/src/material/tabs.dart]: btw we should consider replacing this with a selector that doesn't need a tab controller and just reads out the page information from the nearest PageView.

File: packages/flutter/lib/src/rendering/wrap.dart [hunk 2]
```diff
@@ -63,8 +63,26 @@ class WrapParentData extends ContainerBoxParentDataMixin<RenderBox> {
   int _runIndex = 0;
 }
 
+/// Displays its children in multiple horizontal or vertical runs.
+///
+/// A [RenderWrap] lays out each child and attempts to place the child adjacent
+/// to the previous child in the main axis, given by [direction], leaving
+/// [spacing] space inbeween. If there is not enough space to fit the child,
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/wrap.dart]: inbeween

File: packages/flutter/lib/src/scheduler/ticker.dart [hunk 3]
```diff
@@ -363,6 +363,8 @@ class TickerFuture implements Future<Null> {
     _secondaryCompleter?.completeError(new TickerCanceled(ticker));
   }
 
+  /// A future that resolves when this future resolves or when the ticker is
+  /// canceled.
```
Comment (Hixie) [inline packages/flutter/lib/src/scheduler/ticker.dart]: we should say here that it throws when the ticker is canceled

File: packages/flutter/lib/src/widgets/basic.dart [hunk 4]
```diff
@@ -2284,7 +2284,24 @@ class Expanded extends Flexible {
   }) : super(key: key, flex: flex, fit: FlexFit.tight, child: child);
 }
 
+/// A widget that displays its children in multiple horizontal or vertical runs.
+///
+/// A [Wrap] lays out each child and attempts to place the child adjacent to the
+/// previous child in the main axis, given by [direction], leaving [spacing]
+/// space inbeween. If there is not enough space to fit the child, [Wrap]
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: inbeween

**Comment by Hixie** [comment]
LGTM

**Comment by xster** [comment]
LGTM

**Pull request merged**